### DISCUSSION TITLE
I want detailed variable and interface descriptions

### DISCUSSION THREAD

**Discussion by shentaodeepwave** [pyswmm/pyswmm#558] [Q&A] [answered]
I want detailed descriptions of variables and interfaces, where do I go to find them, for example if I use .flooding what do I get? I want to calculate the total overflow for the four SO nodes in my inp file, how do I do that? Why is the CSO in this example using the cumulate inflow of OF1,OF2,OF3?

**Comment by karosc** [comment, marked as answer]
pyswmm is just a python interface to EPA SWMM. To effectively use pyswmm, you should know the basics of SWMM first. EPA has [extensive docs on SWMM](https://www.epa.gov/water-research/storm-water-management-model-swmm)

That being said, to compute cso, you should sum the total_inflow property of each of your outfalls or the flow property of each of the links just upstream of the outfall. Remember the results are a flow rate, so you'll need to multiply by your timestep to get to volume. 

Node flooding is the water that comes out of the top of a node when the hgl exceeds the ground surface elevation.

**Comment by dickinsonre** [comment]
You can also go to openswmm.org and you can search for past threads about the variable and how to use the variables.  https://www.openswmm.org/Topic/18048/outflow-and-overflow-of-storage
